recipes/tensorboard/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set version = "0.1.6" %}
+{% set build_number = "0" %}
+
+package:
+  name: tensorboard
+  version: {{ version }}
+
+build:
+  # There are two wheels (py2, py3) in pypi, that seem equivalent to me.
+  # So for the time being, we try building a noarch package.
+  # Also see:
+  #  https://github.com/conda-forge/conda-smithy/pull/531
+  noarch: python
+  number: {{ build_number }}
+  entry_points:
+    - tensorboard = tensorboard.main:main
+  script:
+    - pip install --no-deps tensorflow-tensorboard=={{ version }}
+    - cp ${RECIPE_DIR}/LICENSE ${SRC_DIR}  # [not win]
+    # - copy %RECIPE_DIR%/LICENSE %SRC_DIR%  # [win]
```

Comment:
It seems that this is why windows is failing.  If you want to skip windows you need to have 

```yaml
build:
  skip: true  # [win]
```